docs: Example systemd service file

barakplasma opened the pull request:
Thanks @willemstuursma for this project! It's the only fork I could get successfully working on my Raspberry Pi Zero W v1.1
This service file example works for me on Raspbian at least. Feel free to close this PR if you don't want to include an example file.

barakplasma committed:
Example systemd service file

barakplasma committed:
docs: update install instructions

willemstuursma commented:
Good day @barakplasma. Thank you for your compliments and documentation addition. 

I think including the `castblock.service` file in the root would be better than having it in the readme. Could you make that change. 

It would be nice to have one line on why you'd want to have a system.d file.

barakplasma committed:
docs: add reference to service file
lint: remove extra whitespace at end of lines

barakplasma committed:
docs:

barakplasma commented:
Hi @willemstuursma 

I added castblock.service to the root of the git repo. I just added a reference to that file within the README.md and an explanation of why systemd is great to have.

barakplasma committed:
docs:

willemstuursma commented:
Thanks!
